.rustfmt.toml

```diff
@@ -0,0 +1,64 @@
+max_width = 90 # changed
+hard_tabs = false
+tab_spaces = 4
+newline_style = "Auto"
+use_small_heuristics = "Default"
+indent_style = "Block"
+wrap_comments = false
+format_code_in_doc_comments = false
+comment_width = 80
+normalize_comments = true # changed
+normalize_doc_attributes = false
+format_strings = false
+format_macro_matchers = false
+format_macro_bodies = true
+empty_item_single_line = true
+struct_lit_single_line = true
+fn_single_line = false
+where_single_line = false
+imports_indent = "Block"
+imports_layout = "Vertical" # changed
+imports_granularity = "Crate" # changed
+reorder_imports = true
+reorder_modules = true
+reorder_impl_items = false
+type_punctuation_density = "Wide"
+space_before_colon = false
+space_after_colon = true
+spaces_around_ranges = false
+binop_separator = "Front"
+remove_nested_parens = true
+combine_control_expr = false # changed
+overflow_delimited_expr = false
+struct_field_align_threshold = 0
+enum_discrim_align_threshold = 0
+match_arm_blocks = true
+force_multiline_blocks = true # changed
```

Comment:
Ditto; is it desirable to enforce more LOC?

Reply:
Again, better to write readable code(for reviewers) than try to put everything in one line with `if a > b { 24 } else { 42 }`. IDE will collapse it for you in code during navigation, so you will not see it

Reply:
Just to iterate on the reviewability point a little more, more LOC seems like it would make diffs in github easier to follow, as the changes would be more specific and isolated.

Reply:
I really dislike this one, especially for nested match blocks and single-expression closures. It hurts readibility quite a bit, since I'm accustomed to block having multiple expressions.

Reply:
Better to not have nested match blocks=D It will force you to solve them

Reply:
seems like this one isn't so popular, maybe we should revert to default here